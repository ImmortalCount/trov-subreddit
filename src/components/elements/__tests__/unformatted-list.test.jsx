import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'
import UnformattedList from '../unformatted-list'

describe('UnformattedList Component', () => {
    const wrapper = shallow(
        <UnformattedList>
            <li>test</li>
        </UnformattedList>
    )

    it('should render UnformattedList component', () => {
        expect(wrapper.find('li').length).toBe(1)
    })

    it('UnformattedList matches snapshot', () => {
        const modalRender = renderer.create(
            <UnformattedList>
                <li>test</li>
            </UnformattedList>
        ).toJSON()
        expect(modalRender).toMatchSnapshot()
    })
})
