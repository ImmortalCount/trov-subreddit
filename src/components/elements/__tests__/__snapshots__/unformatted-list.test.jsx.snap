// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnformattedList Component UnformattedList matches snapshot 1`] = `
<ul
  className="sc-bdVaJa EFgaU"
>
  <li>
    test
  </li>
</ul>
`;
